src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java

```diff
@@ -140,7 +141,7 @@ public void processElement(DoFn<Long, String>.ProcessContext c) throws Exception
         c.output(String.valueOf(c.element()));
       }
     }).named("toString"));
-    readCountText.apply(TextIO.Write.to(options.getOutput()).named("WriteOutput"));
+    readCountText.apply(TextIO.Write.to(options.getOutput()).named("WriteOutput").withoutSharding());
```

Comment:
Please undo this change and add `--shardBAMReading=false` to the options in the test.